### PR TITLE
Audit accel

### LINKED ISSUE #5023: Audit Ctrl/Accel for keyboard shortcuts
The default shortcuts schema file has about 10 shortcuts that use `Ctrl` rather than `Accel`. Some may be deliberate, but I think many are not. We should go through and determine which ones should be changed to `Accel`.

### PR DESCRIPTION
**Pull request by ian-r-rose**
Fixes #5023. This changes some `Ctrl` shortcuts to `Accel` for more consistency with Mac users. I'd appreciate it if a Mac user could test the binder link to see if things seem to be working correctly.

|  Command  | Old |  New  | Reason | 
|---|---|---|---|
|  `application:activate-next-tab` | `Ctrl Shift ]`  | Unchanged   |  Avoid collision with browser shortcuts on Mac+Chrome (#1681) |
|  `application:activate-previous-tab` |  `Ctrl Shift [` |  Unchanged | Avoid collision with browser shortcuts on Mac+Chrome (#1681)  |
|   `editmenu:redo` | `Ctrl Shift Z`   | `Accel Shift Z`   | Better for Mac users   |
|   `editmenu:undo` | `Ctrl  Z`   | `Accel  Z`   | Better for Mac users   |
|   `filemenu:close-and-cleanup` | `Ctrl Shift Q`   | `Accel  Shift Q`   | Better for Mac users ? |
| `helpmenu:toggle` | `Ctrl Shit H` | Removed | Unused shortcut
| `console:linebreak` | `Ctrl Enter` | `Accel Enter` | Better for mac users, notebook uses `Ctrl Enter` to do something different |
| `notebook:enter-command-mode` | `Ctrl M` (??) | Unchanged | Feature parity with classic |
| `notebook:run-cell` | `Ctrl Enter` | Unchanged | Feature parity with classic |
| `notebook:split-cell-at-cursor` | `Ctrl Shift -` | Unchanged | Feature parity with classic |

## References

#5203 
#1681
## Code changes

* Some new keyboard shortcuts
* Terminals now have a `closeAndCleaner` menu item
* The closeAndCleaner delegator delegates to the application `close` if there is no more specific action to be taken.

## User-facing changes

"Close and Cleanup" now applies more uniformly to widgets.
Some Mac users will see their shortcuts change to `Cmd`.

## Backwards-incompatible changes

None.

### CODE REVIEW

File: packages/mainmenu-extension/src/index.ts [hunk 1]
```diff
@@ -298,6 +298,9 @@ export function createFileMenu(
   const commands = menu.menu.commands;
 
   // Add a delegator command for closing and cleaning up an activity.
+  // This one is a bit difference, in that we consider it enabled
```
Comment (blink1073) [inline packages/mainmenu-extension/src/index.ts]: different*

**Comment by jasongrout** [comment]
Testing this, I noticed that the redo event was never getting triggered on mac. After testing a lot of things, I realized that on mac/firefox, pressing command, shift, and a key was triggering a keydown event with shiftKey set to *false*. To test, you can go to a blank page, open the console, and do `document.addEventListener('keydown', console.log);` and then press, say `Command Shift \`. You can notice that the console says it has a Meta Shift keyboard event, but the shiftKey attribute is false. You can also see this by going to http://nonan.jp/keyevent/input.html and pressing the `Command Shift \` combination and note that shiftKey is set correctly for the meta keydown event, but it is false for `\` keydown event. In fact, in testing this, I'm seeing that for a keydown event for shift and \ and any combination of control, alt, and command, every event shows `event.shiftKey` as true *except* `Command Shift \`

I also tested firefox on windows, and it correctly reported the shiftKey status (though there command is mapped to the OS modifier on that test page).

The end result is that the `Command Shift Z` shortcut doesn't work for me on firefox on mac, but does on Chrome on mac. But this looks like a firefox bug, not a JLab bug.

**Comment by jasongrout** [comment]
> `filemenu:close-and-cleanup` 	`Ctrl Shift Q` 	`Accel Shift Q` 	Better for Mac users ?

Testing this, I noticed that `Cmd Shift Q` is the system-wide Logout command, and `Cmd Ctrl Q` is the system-wide lock screen. So I think this should be reverted to `Ctrl Shift Q`.

**Comment by jasongrout** [comment]
> > `filemenu:close-and-cleanup` 	`Ctrl Shift Q` 	`Accel Shift Q` 	Better for Mac users ?
> 
> Testing this, I noticed that `Cmd Shift Q` is the system-wide Logout command, and `Cmd Ctrl Q` is the system-wide lock screen. So I think this should be reverted to `Ctrl Shift Q`.

Actually, for a mac, Cmd W is close tab and Cmd Shift W is close window, whereas Q is associated with closing an application or the system (locking or logging out), so from that perspective, `Ctrl Shift W` or even `Ctrl W` probably makes more sense on a mac. However, I can also see the argument for Ctrl Shift Q for compatibility for JLab across operating systems.

**Comment by ian-r-rose** [comment]
Sounds good. @jasongrout Do think the `Cmd-Shit` issue you saw is bad enough to merit some additional changes of `Cmd` to `Ctrl` on macs?

**Comment by jasongrout** [comment]
> Sounds good. @jasongrout Do think the `Cmd-Shift` issue you saw is bad enough to merit some additional changes of `Cmd` to `Ctrl` on macs?

Possibly. We can do that on another issue. I'd love if someone on a mac can confirm too to make sure I'm not crazy (or my browser).

**Comment by jasongrout** [comment]
Thanks!

**Comment by jasongrout** [comment]
> Sounds good. @jasongrout Do think the `Cmd-Shift` issue you saw is bad enough to merit some additional changes of `Cmd` to `Ctrl` on macs?

Turns out that creating a new firefox profile made things start working for me, and things worked fine on two other macs we were testing, so I think this was just my issue.

**Pull request merged**